Sources/SimpleCurrencyConverter/SimpleCurrencyConverter.swift

```diff
@@ -1,6 +1,141 @@
-public struct SimpleCurrencyConverter {
-    public private(set) var text = "Hello, World!"
+import Foundation
 
-    public init() {
+public enum Currency: String {
+    case JPY = "JPY"
+    case USD = "USD"
+    case GBP = "GBP"
+    case EUR = "EUR"
+}
+
+public class SimpleCurrencyConverter: CurrencyConverterProtocol {
+
+    private var exchangeRatesAPIKey: String?
+    private var networkManager: ExchangeRatesNetworkManager?
+    private static let dependencyContainer = DependencyContainer()
```

Comment:
@ts-dzmitry-kurski I don't think that `dependencyContainer` needs to be `static` here as it needs to be created only when `SimpleCurrencyConverter` is instantiated. Can you confirm?

Reply:
@malayli It should be static because `dependencyContainer` instance is used in static methods like `shared()` and `setup()`.

Reply:
I have this kind of error `Instance member 'dependencyContainer' cannot be used on type 'SimpleCurrencyConverter'` in case of non-static variable.

Reply:
@ts-dzmitry-kurski Ok strange, it should work. Let's ignore this issue in this PR.